packages/plugins/typescript/typescript/tests/typescript.spec.ts

```diff
@@ -287,6 +291,7 @@ describe('TypeScript', () => {
 
       expect(result).toBeSimilarStringTo(`
         export type MyType = {
+          __typename?: "MyType",
```

Comment:
this should be readonly __typename

Reply:
It could, we have a configuration for immutable types, be default they are not.